spec/policies/case_assignment_policy_spec.rb

```diff
@@ -67,11 +67,12 @@
     context "when an admin" do
       context "in the same organization" do
         it "allows user to show or hide contacts" do
-          is_expected.to permit(casa_admin, case_assignment)
+          is_expected.to permit(casa_admin, case_assignment_inactive)
         end
       end
       context "in a different organization" do
         it "does not allow user to show or hide contacts" do
+          other_case_assignment.update(active: false)
```

Comment:
This is a good call. Otherwise an admin might not have perms because the case is inactive rather than because the case belongs to another org.